sources/ReplicantWindow.cpp

```diff
@@ -7,16 +7,20 @@
  */
 
 
+#include <Catalog.h>
 #include <ControlLook.h>
 #include <LayoutBuilder.h>
 
 #include "DropZoneTab.h"
 #include "ReplicantWindow.h"
 
+#undef B_TRANSLATION_CONTEXT
+#define B_TRANSLATION_CONTEXT "ReplicantWindow"
+
 
 ReplicantWindow::ReplicantWindow(BRect frame)
 	:
-	BWindow(BRect(0, 0, 120, 100), ("Resize"),
+	BWindow(BRect(0, 0, 120, 100), (B_TRANSLATE("Resize")),
```

Comment:
We have to make sure nobody translates the window title into something long, as that prevents the replicant from getting narrower. I think we may get away with a titleless window, as you suggested. To avoid having a blank window under "Filer" in the Deskbar, I suggest this:
```
BWindow(BRect(0, 0, 120, 100), (B_TRANSLATE("")),
	B_DOCUMENT_WINDOW_LOOK, B_FLOATING_APP_WINDOW_FEEL,
	B_NOT_ZOOMABLE | B_AUTO_UPDATE_SIZE_LIMITS)
```
As FLOATING window it doesn't appear in the Deskbar. Would be nice to have a smaller yellow tab, but then the resize corner would be very small as well, which defies the purpose of the window... :)